linux/daemon/helper.sh

```diff
@@ -189,25 +140,16 @@ get_fwmark() {
 }
 
 remove_firewall() {
-	if type -p nft >/dev/null; then
```

Comment:
@bakulf just to confirm i'm removing nft support.

Reply:
ok!